runtime/internal/BUILD.bazel

```diff
@@ -0,0 +1,23 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")
+
+package(default_visibility = ["//visibility:public"])
```

Comment:
Could you make the visibility of the package consistent with the one by go compiler?
i.e. it looks to be better to limit the default visibility to `//runtime:__subpackages__`.

BTW, as the original author of Gazelle, I'm sorry -- it should have been done by gazelle. ;-)

Reply:
You are excused hehe. Thanks for the Gazelle, it saves tons of time daily for me.
I also made changes to other package visibilities as I saw fit.